tide/master/control/DisplayGroupController.cpp

```diff
@@ -120,21 +127,56 @@ void DisplayGroupController::unfocus( const QUuid& id )
         return;
 
     _group.removeFocusedWindow( window );
+    _readjustToNewZoomLevel( *window );
+    window->setControlsVisible( false );
+
     updateFocusedWindowsCoordinates();
+}
 
-    // Make sure the window dimensions are re-adjusted to the new zoom level
-    ContentWindowController{ *window, _group }.scale( window->center(), 0.0 );
+void DisplayGroupController::focusSelected()
+{
+    unfocusAll(); // ensure precondition, but should already be empty
+
+    auto focusedWindows = ContentWindowSet{};
+
+    for( const auto& window : _group.getContentWindows( ))
+        if( window->getControlsVisible( ))
```

Comment:
that's right, maybe even rename "controlsVisible" to "selected" entirely, as this is becoming just a side-effect.